### DISCUSSION TITLE
Web News API giving error

### DISCUSSION THREAD

**Discussion by Carvalho286** [community/community#143237] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Is News API down for the moment?

**Comment by jdtdm09** [comment]
Restart bro

**Reply by samus-aran** [reply]
@Carvalho286  and @jdtdm09 
Bros... try and be less subtle next time 😉

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
